packages/components/src/composite/index.ts

```diff
@@ -17,3 +17,6 @@ export {
 	CompositeItem,
 	useCompositeState,
 } from 'reakit';
+
+/* eslint-disable-next-line no-restricted-imports */
+export type { CompositeStateReturn as CompositeState } from 'reakit';
```

Comment:
Let's delete this export, since it's not used anywhere 🧹

Reply:
It's used in `circular-option-picker/types.ts`...

```ts
export type CircularOptionPickerCompositeState = CompositeState;
export type CircularOptionPickerContextProps =
	| { isComposite?: false; baseId?: string }
	| ( { isComposite: true } & CircularOptionPickerCompositeState );
```

Reply:
My bad, I must have missed that — feel free to ignore my comment.